### DISCUSSION TITLE
Return User ID within Auth/Login

### DISCUSSION THREAD

**Discussion by brunoribei-ro** [directus/directus#4988] [Archive]
Is it possible to get user id included in auth/login response body?

**Comment by alexvdvalk** [comment]
The user id is held within the JWT token. You can use a package like https://www.npmjs.com/package/jwt-decode to decode the token that you receive and that will return  a JSON object like this:

````JSON
{
  "id": "YOUR USER ID",
  "role": "gdfg",
  "app_access": 1,
  "admin_access": 1,
  "iat": 234234,
  "exp": 45676,
  "iss": "directus"
}
